libs/clients/syslumenn/src/lib/syslumennClient.types.ts

```diff
@@ -129,9 +129,12 @@ export enum PersonType {
 
 export interface MortgageCertificate {
   contentBase64: string
+  apiMessage?: string | undefined
 }
 
 export interface MortgageCertificateValidation {
+  propertyNumber: string
+  isFromSearch?: boolean | undefined
```

Comment:
The question mark means that it could also be undefined so it's not necessary to say it's undefined, just pointing that. If you think it's better to be more explicit this way that's fine too.

Reply:
Ah ok, will fix this